diff --cc Source/Particles/Collision/BackgroundMCC/BackgroundMCCCollision.cpp
index 1938291b3,b9763b2e2..000000000
--- a/Source/Particles/Collision/BackgroundMCC/BackgroundMCCCollision.cpp
+++ b/Source/Particles/Collision/BackgroundMCC/BackgroundMCCCollision.cpp
@@@ -7,8 -7,8 +7,10 @@@
  #include "BackgroundMCCCollision.H"
  
  #include "ImpactIonization.H"
 +#include "Particles/Collision/BinaryCollision/BinaryCollisionUtils.H"
 +#include "Particles/Collision/BinaryCollision/TwoProductUtil.H"
+ #include "IonImpactIonization.H"
+ #include "Particles/Algorithms/KineticEnergy.H"
  #include "Particles/ParticleCreation/FilterCopyTransform.H"
  #include "Particles/ParticleCreation/SmartCopy.H"
  #include "Utils/Parser/ParserUtils.H"
diff --cc Source/Particles/Collision/ScatteringProcess.H
index c111a0123,62086b6ec..000000000
--- a/Source/Particles/Collision/ScatteringProcess.H
+++ b/Source/Particles/Collision/ScatteringProcess.H
diff --git a/.gitignore b/.gitignore
index 8e6963db0..a98977dad 100644
--- a/.gitignore
+++ b/.gitignore
@@ -77,3 +77,15 @@ WARP.md
 .DS_Store
 .AppleDouble
 .LSOverride
+
+
+
+INSTALL.md
+Source/Particles/Collision/BackgroundMCC/BackgroundMCCCollision.H.orig
+Source/Particles/Collision/BackgroundMCC/BackgroundMCCCollision.cpp.orig
+Source/Particles/Collision/BackgroundMCC/IonImpactIonization.H
+Testing/
+install/
+patch/
+upgrade.sh
+
